errors/src/test/java/org/odk/collect/errors/RobolectricApplication.kt

```diff
@@ -0,0 +1,11 @@
+package org.odk.collect.errors
+
+import android.app.Application
+import org.odk.collect.strings.localization.LocalizedApplication
+import java.util.Locale
+
+class RobolectricApplication : Application(), LocalizedApplication {
+
+    override val locale: Locale
+        get() = this.resources.configuration.locales[0]
```

Comment:
I see we did this in `geo` as well. Do you think we should make `LocalizedActivity` fallback to the default `Locale` instead of relying on the underlying application being a `LocalizedApplication`? It feels like that would match what we do in `getLocalizedString` - just fallback to using the default Locale rather than exploding.

Reply:
Yeah that makes sense, fixed.